repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -754,3 +754,9 @@ export interface MultiTap {
 export interface MultiTapPricing {
     tapDetails: MultiTap;
 }
+
+export interface AdditionalPricing {
+    additionalDiscounts: string;
```

Comment:
I dont think we need additionalDiscounts on this row - we should either store them in the session as ```
{  pricingStructureStart: string;
    structureDiscount: string; }
```

or store nothing in the session if we dont.

Reply:
And if it is in the session, we know they said yes. And if its not, we know they said no.